### ISSUE TITLE
git blame -Rajoy

### ISSUE THREAD

**Issue by saretdm** [voghDev/git-pushdemont#18]

**Comment by isdelrey** [comment]
Yes, I have been debugging this. The push reversion is due to a corrupt module: 'rajoy'. It's a dependency from the 'Spain' module. Maybe we should remove the Spain module in the next commit. That might bring problems with main dependency Constitution. What do you guys propose?
